modal_generator.py

```diff
@@ -197,7 +197,9 @@ def get_create_config_file_modal():
     """
     ret = dbc.Modal(
         [
-            dbc.ModalHeader("Create config file"),
+            dbc.ModalHeader(
+                H5("Create config file")
+            ),
```

Comment:
Can you change this back to what it was before? I don't think the other Modal Headers have H5.